Apps/Examples/Examples/All Examples/CustomLocationProviderExample.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import MapboxMaps
+
+@objc(CustomLocationProviderExample)
+internal class CustomLocationProviderExample: UIViewController, ExampleProtocol {
```

Comment:
```suggestion
final class CustomLocationProviderExample: UIViewController, ExampleProtocol {
```

* Mark classes final if they're not designed to be subclassed.
* In an app module, it's less useful to be explicit about internal.